### PR TITLE
Use common payment method

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Use `paymentMethod` from `stripes-acq-components`

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/common/constants/constants.js [hunk 1]
```diff
@@ -30,27 +35,10 @@ export const INVOICE_STATUSES_OPTIONS = [
...
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.cash" />, value: PAYMENT_METHOD.cash },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.creditCard" />, value: PAYMENT_METHOD.creditCard },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.eft" />, value: PAYMENT_METHOD.eft },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.depositAccount" />, value: PAYMENT_METHOD.depositAccount },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.physicalCheck" />, value: PAYMENT_METHOD.physicalCheck },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.bankDraft" />, value: PAYMENT_METHOD.bankDraft },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.internalTransfer" />, value: PAYMENT_METHOD.internalTransfer },
-  { label: <FormattedMessage id="ui-invoice.paymentMethod.other" />, value: PAYMENT_METHOD.other },
-];
+export const PAYMENT_METHOD_OPTIONS = Object.values(PAYMENT_METHOD).map(method => ({
```
Comment (aliaksei-chumakou) [inline src/common/constants/constants.js]: isn't it the same in different modules? I thought you've added this to acq-components also

File: test/bigtest/network/factories/invoice.js [hunk 2]
```diff
@@ -14,5 +11,5 @@ export default Factory.extend({
   total: () => Number.parseFloat(faker.finance.amount(90, 1000, 2)),
   currency: 'USD',
   approvalDate: faker.date.past,
-  paymentMethod: PAYMENT_METHOD.cash,
+  paymentMethod: 'Cash',
```
Comment (aliaksei-chumakou) [inline test/bigtest/network/factories/invoice.js]: maybe import from acq-components works better here

File: src/common/constants/constants.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/common/constants/constants.js]: I added options for `<FieldSelect>` where we use `labelId` instead `label`

Comment (aliaksei-chumakou) [reply src/common/constants/constants.js]: But you have it in stripes-acq-components, no? https://github.com/folio-org/stripes-acq-components/blob/1ffbf932fce0ed41e28bf1486953e1946e9c449a/lib/constants/paymentMethod.js#L37

**Comment by aliaksei-chumakou** [comment]
@AndreiShumski2  could you please remove fixed eslint things? They useful only until they're fixed, after they just litter the screen space

File: src/common/constants/constants.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/common/constants/constants.js]: [acqCheckBoxFilter](https://github.com/folio-org/stripes-acq-components/pull/71) was extended to use these options

**Pull request merged**